### ISSUE TITLE
Get unexpected error when deleting message with client SDK record API

### ISSUE THREAD

**Issue by Steven-Chan** [SkygearIO/chat#148]
The app is running on skygear cloud, version is v1.1.0

I am not sure if deleting message without calling chat lambdas is allowed, but the following message is unexpected to me.

```
{ [Error: message_before_delete_handler() missing 1 required positional argument: 'conn']
  message: 'message_before_delete_handler() missing 1 required positional argument: \'conn\'',
  code: 10000,
  info: null }
```

**Comment by howawong** [comment]
No, it is not allowed to delete message via record delete API. Please call lambda instead. But the `TypeError`exception is not expected neither.

The following is the expected HTTP response. 

```json
{
    "result": [
        {
            "_id": "message/017D4741-08A2-40EA-B6CF-67BF004DF190",
            "_type": "error",
            "code": 111,
            "info": {},
            "message": "This operation is not supported.",
            "name": "NotSupported"
        }
    ]
}
```

**Comment by rickmak** [comment]
I merged #152, but I think the message should be something like following:


```
{
    "result": [
        {
            "_id": "message/017D4741-08A2-40EA-B6CF-67BF004DF190",
            "_type": "error",
            "code": 111,
            "info": {},
            "message": "This operation is not supported. Please call lambda `chat:delete_message`",
            "name": "NotSupported"
        }
    ]
}
```
